components/dependency_injection/autowiring.rst

```diff
@@ -308,7 +308,10 @@ and a Twitter client using it:
         services:
             rot13_transformer:
                 class:            Acme\Rot13Transformer
-                autowiring_types: Acme\TransformerInterface
+
+            Acme\TransformerInterface:
+                alias: rot13_transformer
+                public: false
```

Comment:
why bother with `public: false`? Let's show the short `Acme\TransformerInterface: '@ rot13_transformer'` instead :)